api/proto/dop/projectpipeline/projectpipeline.proto

```diff
@@ -86,8 +86,82 @@ service ProjectPipelineService {
       path : "/api/project-pipeline/definitions/{pipelineDefinitionID}",
     };
   }
+
+  rpc Run(RunProjectPipelineRequest)
+          returns(RunProjectPipelineResponse) {
+    option(google.api.http) = {
+      post : "/api/project-pipeline/definitions/{pipelineDefinitionID}/actions/run",
```

Comment:
`/api/pipeline-definition/{}/actions/run` is better

Reply:
![image](https://user-images.githubusercontent.com/28723047/161884534-830bcf8c-9318-4f77-9577-1479220357c9.png)

No better. The pipeline layer is prefixed with this API